### PR TITLE
Remove report from search serializer

### PR DESCRIPTION
**Pull request by jonasdeluna**
In the future we may want to allow searching for meeting reports you are a part of but this is a quick fix for the security issue.

Resolves: ABA-563

### CODE REVIEW

File: lego/apps/meetings/search_indexes.py [hunk 1]
```diff
@@ -10,13 +10,12 @@ class MeetingModelIndex(SearchIndex):
     result_fields = (
```
Comment (norbye) [inline lego/apps/meetings/search_indexes.py:10]: Have you tried changing the `queryset` to something along the lines of `Meeting.objects.filter(invited__includes=user.id)` (or some other variation that might be more correct:p)?

**Review by norbye — APPROVED** [review]
Would be awesome if you consider playing with the queryset before going for this approach - but I'll leave it as approved so it's up to you

**Comment by norbye** [comment]
Whops just noticed that this was set to canceled in linear - so maybe not heheh - just close the PR if it's not happening

**Comment by danielyanghansen** [comment]
As per [ABA-563](https://linear.app/abakus-webkom/issue/ABA-563) and what we discuesed on our meeting like a month ago, this can be closed

**Pull request closed without merging**